macro/src/expand.rs

```diff
@@ -1145,10 +1145,17 @@ fn expand_rust_function_shim_impl(
         let ret = if let Type::Future(fut) = ret {
             let span = sig.ret.span();
             let output = &fut.output;
+            let lifetimes: Vec<_> = sig.generics.lifetimes().map(|lt| quote!(#lt)).collect();
+            let lifetimes = if lifetimes.is_empty() {
+                quote!()
+            } else {
+                assert_eq!(lifetimes.len(), 1, "workerd-cxx: expected single lifetime (todo: do we need to support them?)");
```

Comment:
Need to support "them"? Do you mean multiple lifetimes?

Reply:
reworded